examples/cart/index.js

```diff
@@ -158,13 +160,37 @@ $(function() {
   ============================================================ */
   function openCart() {
     $('.cart').addClass('js-active');
+    $('.overlay').addClass('js-active');
   }
 
   /* Close Cart
   ============================================================ */
   function closeCart() {
-    $('.cart .btn--close').click(function () {
-      $('.cart').removeClass('js-active');
+    $('.cart').removeClass('js-active');
+    $('.overlay').removeClass('js-active');
+  }
+
+  /* Close Cart Handler
+  ============================================================ */
+  function closeCartHandler() {
+    $('.cart .btn--close').click(closeCart);
```

Comment:
on is still preferred, it's just moved to the jQuery-specific guide: https://github.com/Shopify/javascript/blob/master/jquery/README.md